src/openMVG/sfm/utils/alignment.hpp

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+#include "openMVG/sfm/sfm_data.hpp"
+#include "openMVG/geometry/rigid_transformation3D_srt.hpp"
+
+namespace openMVG {
+namespace sfm {
+
+inline void getCommonViews(
+  const SfM_Data & sfmDataA,
+  const SfM_Data & sfmDataB,
+  std::vector<IndexT>& outIndexes
+  )
+{
+  for(const auto& viewA: sfmDataA.GetViews())
+  {
+    if(sfmDataB.GetViews().find(viewA.first) != sfmDataB.GetViews().end())
+    {
+      outIndexes.push_back(viewA.first);
+    }
+  }
+}
+
+/**
+ * @brief Compute a 5DOF rigid transform between the two set of cameras.
+ *
+ * @param[in] sfmDataA
+ * @param[in] sfmDataB
+ * @param[out] out_S: output scale factor
+ * @param[out] out_R: output rotation 3x3 matrix
+ * @param[out] out_t: output translation vector
+ * @return true if it finds a similarity transformation
+ */
+inline bool computeSimilarity(
+  const SfM_Data & sfmDataA,
+  const SfM_Data & sfmDataB,
+  double * out_S, Mat3 * out_R, Vec3 * out_t)
+{
```

Comment:
damn I got corrupted by the french spelling! :-)
Anyway I was more suggesting to move the implementation in a `.cpp` file